src/Controller/HomeController.php

```diff
@@ -11,4 +11,9 @@ public function index(): string
     {
         return $this->twig->render('Home/index.html.twig');
     }
+
+    public function rules(): string
```

Comment:
since rules is not part of the Home/index page, please create a Rules/RulesController class with an index method